packages/dev/core/src/Buffers/buffer.ts

```diff
@@ -1026,25 +1026,16 @@ export class VertexBuffer {
                 return new Float32Array(data, byteOffset, count);
             } else {
                 const offset = data.byteOffset + byteOffset;
-                if (forceCopy) {
-                    const result = new Float32Array(count);
-                    const source = new Float32Array(data.buffer, offset, count);
-
+                const wrongAlign = (offset & 3) !== 0;
+                if (forceCopy || wrongAlign) {
+                    if(wrongAlign) console.warn("Float array must be aligned to 4-bytes border")

```

Comment:
Please use Tools.Warn instead of console.warn

Reply:
Logger.Warn is probably the best option.